src/models/tick-model/ticks/__tests__/tick-helper.spec.js

```diff
@@ -190,26 +186,27 @@ describe('getSize', () => {
   it('should return correct size if prelayout is false and dimension is width for nomal data', () => {
     chartModel.query.isPrelayout.returns(false);
     dimension = 'width';
+    chart.component.withArgs('point-component').returns({ rect: { computedPhysical: { width: 80, height: 40 } } });
     expect(create()).to.equal(80);
   });
 
   it('should return correct size if prelayout is false and dimension is height for nomal data', () => {
     chartModel.query.isPrelayout.returns(false);
     dimension = 'height';
+    chart.component.withArgs('point-component').returns({ rect: { computedPhysical: { width: 80, height: 40 } } });
+    chart.component.withArgs('heat-map').returns({ rect: { computedPhysical: { width: 0, height: 0 } } });
     expect(create()).to.equal(40);
   });
 
-  it('should return correct size if prelayout is false and dimension is width for bin data', () => {
-    chartModel.query.isPrelayout.returns(false);
-    dataHandler.getMeta.returns({ isBinnedData: true });
-    dimension = 'width';
-    expect(create()).to.equal(100);
-  });
+  // it('should return correct size if prelayout is false and dimension is width for bin data', () => {
+  //   chartModel.query.isPrelayout.returns(false);
+  //   dimension = 'width';
+  //   expect(create()).to.equal(100);
+  // });
```

Comment:
These should either be fixed or removed (?)

Reply:
Good eyes. Yes, they should be fixed. I forgot about them while dealing with another test file.